app/jsx/pages/BrowsePage.jsx

```diff
@@ -77,11 +165,15 @@ class BrowseJournals extends React.Component {
 //       :ancestor_unit=>["ucla", "ucsc"],
 //       :is_active=>true}
   getVisibleJournals(journals, isActive, campusID) {
-    return journals.map(function(j, i) {
-      return (j['is_active'] == isActive || isActive == "") &&
+    let foundOne = false
+    let r = journals.map(function(j, i) {
```

Comment:
Wireframes indicate ability to choose archived items as well as actively publishing. http://ux.cdlib.org/jkratz/eScholarship/wireframes/eScholarship_2016-12-6/#g=1&p=browse_journals